rosco-web/src/main/groovy/com/netflix/spinnaker/rosco/endpoint/StatusEndpoint.groovy

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Please name the package `com.netflix.spinnaker.rosco.endpoints` to be consistent with its sibling `com.netflix.spinnaker.rosco.controllers` package.